draft-sharabayko-mops-srt.md

```diff
@@ -2086,15 +2086,583 @@ and RTTVar are updated by the socket based on algorithms for the sender
+SRT's sender buffer and, as a result, delayed causing their too late
+arrival at the decoder and possible drops by the receiver.
+
+A proposed solution is to have SRT take the input rate configuration
+from the encoder, measure the actual input, and use whichever value
+is larger (HYBRIDBW mode). Combining the two methods overcomes
+the deficiencies of each.
+
+The following table shows the summary of modes, variables need to be
+set (✓) or ignored (-) and formula for MAX_BW calculation for a
+particular mode:
+
+~~~
+| Mode / Variable | MAX_BW | INPUT_BW | OVERHEAD |
+| --------------- | ------ | -------- | -------- |
+| MAXBW           | ✓      | -        | -        |
+| INPUTBW         | -      | ✓        | ✓        |
+| ESTINPUTBW      | -      | -        | ✓        |
+| HYBRIDBW        | -      | ✓        | ✓        |
+~~~
+
+### SRT's Default LiveCC Algorithm {#default-liveCC}
+
+The main goal of the SRT's default LiveCC algorithm is to adjust the
+minimum allowed packet sending period PKT_SND_PERIOD (and, as a result,
+the maximum allowed sending rate) during transmission based on the
+average packet payload size AvgPayloadSize and maximum bandwidth
+MAX_BW.
+
+On the sender side, there are three events LiveCC algorithm reacts to:
+on sending a data packet, on an acknowledgement (ACK) packet reception,
+and on a timeout event as described below.
+
+On sending a data packet (either original, or retransmitted), update
+the value of average packet payload size AvgPayloadSize:
+
+~~~
+AvgPayloadSize = AvgPayloadSize * 0.875 + PacketPayloadSize * 0.125
+~~~
+
+where PacketPayloadSize is the payload size of a sent data packet, in bytes;
+the initial value of AvgPayloadSize is equal the maximum allowed packet
+payload size which can't be larger than 1456 bytes.
+
+On an acknowledgement (ACK) packet reception:
+
+Step 1. Calculate SRT packet size pktsize as the sum of average payload size
+AvgPayloadSize and SRT header ({{packet-structure}}), in bytes.
+
+Step 2. Calculate the minimum allowed packet sending period PKT_SND_PERIOD
+as:
+
+~~~
+PKT_SND_PERIOD = pktsize * 1000000 / MAX_BW
+~~~
+
+where MAX_BW is the configured maximum bandwidth which limits the
+bandwidth usage by SRT, in bytes per second; PKT_SND_PERIOD is
+measured in microseconds.
+
+On a retransmission timeout (RTO) event, follow the same steps as
+described in "On an acknowledgement (ACK) packet reception" part.
+
+RTO is the time that an acknowledgement is expected after a data packet
+is sent out. If there is no ACK after this amount of time elapsed, a
+timeout event is triggered. Since SRT only acknowledges every SYN time
+({{packet-acks}}), the value of retransmission timeout is
+defined as follows:
+
+~~~
+RTO = RTT + 4 * RTTVar + 2 * SYN
+~~~
+
+where RTT is the round-trip time estimation, in microseconds, and
+RTTVar is the variance of RTT estimation, in microseconds, reported
+by the receiver and smoothed at the sender side
```

Comment:
smoothed?

Reply:
Currently, the EWMA is applied twice: 1. at the receiver side, then we send this value within an ACK packet; 2. at the sender side: we extract RTT from ACK and do EWMA again, that's why I writing here smoothed at the receiver side.

Reply:
See [this comment](https://github.com/Haivision/srt-rfc/pull/66#discussion_r492836186) also.